lib/index.js

```diff
@@ -533,39 +366,41 @@ async function hyperlink({
                             follow = true;
                             relation.to.stopProcessing = true;
                         } else {
-                            relation.to.check = true;
+                            metadataOnly = true;
                         }
                     }
                 } else if (/^(?:JavaScript|Css)Source(?:Mapping)Url$/.test(relation.type)) {
                     if (followSourceMaps) {
                         follow = true;
                     } else {
-                        relation.to.check = true;
+                        metadataOnly = true;
                     }
                 } else if (['SourceMapFile', 'SourceMapSource'].includes(relation.type)) {
                     if (followSourceMaps) {
-                        relation.to.check = true;
+                        metadataOnly = true;
                     }
                 } else {
                     follow = true;
                 }
 
-                if (follow) {
+                if (follow || metadataOnly) {
                     if (assetTypesWithoutRelations.includes(relation.to.type)) {
                         // If we are handling local file-urls, follow but mark as end-of-line in processing
                         if (!recursive && relation.from.protocol === 'file:' && relation.to.protocol === 'file:') {
```

Comment:
This branch can probably die if assetgraph handles the loading

Reply:
Fixed in fbaa61bf010405488b76a656274644c502cac7e0